requirements/base.txt

```diff
@@ -1,5 +1,5 @@
-arrow>=0.12,<0.13
+arrow>=0.14,<0.15
 cherrypy>=17.3,<17.4
 requests>=2.20.0
 requests-oauth>=0.4.1,<0.5
-requests-oauthlib>=1.0,<1.1
+requests-oauthlib>=1.2,<1.3
```

Comment:
Can we leave the lower end at 1.0 or does this change break things with < 1.2?

Reply:
I believe we can't use 1.1 as 1.2 introduced a breaking change to only support oauthlib 3.0 https://github.com/requests/requests-oauthlib/commit/1495328dea94b38e3675476cb889e2e3a038355b#diff-d9ae695c8119e58c2ec435b6deae2d53